homeassistant/components/group/__init__.py

```diff
@@ -562,12 +567,29 @@ async def async_create_group(
             ENTITY_ID_FORMAT, object_id or name, hass=hass
         )
 
-        # If called before the platform async_setup is called (test cases)
-        if (component := hass.data.get(DOMAIN)) is None:
-            component = hass.data[DOMAIN] = EntityComponent(_LOGGER, DOMAIN, hass)
+        return group
+
+    @staticmethod
```

Comment:
```suggestion
    @classmethod
```

Reply:
I had to switch it back since it was called in the services as well and the change started to spiral